_data/navigation.yml

```diff
@@ -107,13 +107,13 @@ concepts:
 tutorials:
   general: 
     title: "General service publishing tutorials"
-    url: "/tutorials/index"
+    url: "/tutorials/"
```

Comment:
all of them have the same URL

Reply:
Yup, needs to be cleaned up when we update the tutorials section